packages/api/template/config.yml

```diff
@@ -14,11 +14,22 @@ default:
     version: '6.0'
     instanceCount: 1
     instanceType: t2.small.elasticsearch
-    volumeSize: 35 
+    volumeSize: 35
 
   apis:
     - name: stac
 
+  stacsnsfeed:
+    cbersfeed:
+      name: CBERS
+      topic: arn:aws:sns:us-east-1:769537946825:cbers-2-stac-CBERSSTACItemTopic-N0MZUA5EIQC9
```

Comment:
'name' here could be 'cbersfeed', but I did not know how to reference the group name in kes. It works for items in 'lambda' group (simply using {{name}} without explicit definition) but did not work for the proposed stacsnsfeed group. If using the group name is possible we could remove the 'name' parameter.